module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -1982,15 +1992,17 @@ protected function getOAuth2Token($renew = false)
      */
     protected function getItemStatusesForBiblio($id, $patron = null)
     {
-        $result = $this->makeRequest(
-            [
-                'path' => [
-                    'v1', 'contrib', 'kohasuomi', 'availability', 'biblios', $id,
-                    'search',
-                ],
-                'errors' => true,
-            ]
-        );
+        $requestParams = [
+            'path' => [
+                'v1', 'contrib', 'kohasuomi', 'availability', 'biblios', $id,
+                'search',
+            ],
+            'errors' => true,
+        ];
+        if ($this->includeSuspendedHoldsInQueueLength) {
+            $requestParams['query']['include_suspended_in_hold_queue'] = '1';
```

Comment:
Since the query key is not previously defined, would it be better to say `$requestParams['query'] = ['include_suspended_in_hold_queue' => '1'];` or to set it to an empty array above? I feel like there may be some scenario somewhere where setting multiple nested undefined keys might trigger a notice, though maybe I'm being oversensitive.

Reply:
I added setting of the element in the array above so that any additional query params can be added later without modifying current code.